packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.module.scss

```diff
@@ -6,6 +6,7 @@
     display: flex;
     margin: 0;
     max-width: 100%;  // Needed for IE10 truncation
+    overflow-x: hidden;
```

Comment:
Any concerns about clipping content?

Reply:
Yes. Thanks @StephenEsser - concerns for clipping was the reason this style was originally removed. Now by adding it, it does resolve the situations described in #738, but  now by adding `overflow-x: hidden`, it also now forces `overflow-y: scroll` so if the content is taller for any reason than it's bounding container, it may create vertical scrollbars rather than overflowing, which is unintended and not desired. 

We hope this only to be a short term resolution, as the consumer can also protect against creating the issue with their implementation as described in https://github.com/cerner/terra-clinical/issues/738#issuecomment-781457580, and we will revisit this again as future enhancements are made.